mon-pix/app/utils/session-storage-key.js

```diff
@@ -0,0 +1,18 @@
+export class SessionStorageKey {
```

Comment:
Super l’idée de cette classe :heart: 

Comme l’idée est de stocker une clé+valeur (c’est à dire une _« entry »_, par exemple comme https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) dans les sessionStorage du navigateur, que diriez-vous du nommage suivant ? 

```suggestion
export class SessionStorageEntry {
```

Reply:
Est ce que un service plutôt qu'une classe à instancier à chaques utilisation ne serait pas plus pratique ? Sinon pareil, j'aime beaucoup l'idée :)

Reply:
Renommage de la classe effectué